doc/user_guide/interactions.rst

```diff
@@ -784,36 +791,6 @@ where the user can choose the colors of the chart interactively:
         color_usa, color_europe, color_japan
     )
 
-.. _encoding-channel-binding:
```

Comment:
I moved this to the bottom because I find the legend and scale examples less complicated so they could go first